db/migrate/20161019034559_add_jenkins_email_committers_to_stage.rb

```diff
@@ -0,0 +1,5 @@
+class AddJenkinsEmailCommittersToStage < ActiveRecord::Migration[5.0]
+  def change
+    add_column :stages, :jenkins_email_committers, :boolean
```

Comment:
this is a 3-state boolean atm with true/false/nil -> add `default: false, null: false`